airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml

```diff
@@ -12,10 +12,10 @@ tests:
   discovery:
     - config_path: "secrets/config.json"
       backward_compatibility_tests_config:
-        disable_for_version: "1.0.2"
+        disable_for_version: "2.0.0"
     - config_path: "secrets/connected_account_config.json"
       backward_compatibility_tests_config:
-        disable_for_version: "1.0.2"
+        disable_for_version: "2.0.0"
```

Comment:
what's the customer impact of this change?

Reply:
schema was changes, and as far I understood, this breaking change should be reflected as major version update.